src/PowerShellEditorServices.Hosting/Configuration/HostLogger.cs

```diff
@@ -27,9 +30,14 @@ public enum PsesLogLevel
     }
 
     /// <summary>
-    /// A logging front-end for host startup allowing handover to the backend
-    /// and decoupling from the host's particular logging sink.
+    /// A logging front-end for host startup allowing handover to the backend and decoupling from
```

Comment:
This was a semantic line break before; the newline comes before "and" to minimise enjambment.

Reply:
I'm literally just using the VS Code rewrap extension 😅